src/main/webapp/app/home/home.component.ts

```diff
@@ -0,0 +1,79 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Inject, OnInit, } from '@angular/core';
```

Comment:
Well to be honest, this solution of using primitive properties not observables popped into my mind first so that’s why I implemented it. I believe, as you mentioned, observables would be a better route to take. Updates are available on the next commit.